runtime/lib/backend/null_qubit/InstructionStrBuilder.hpp

```diff
@@ -0,0 +1,383 @@
+
```

Comment:
codefactor could be complaining about this line, sometimes (like half of the time) it reports wrong line numbers for extra blank lines : )

Reply:
I just removed this line. But it still fails :(

Reply:
was a good catch though! thanks!